src/Fmt_ast.ml

```diff
@@ -799,7 +799,11 @@ and fmt_pattern c ?pro ?parens ({ctx= ctx0; ast= pat} as xpat) =
       fun k ->
         Cmts.fmt c ~pro:spc ppat_loc
         @@ Cmts.fmt c ~pro:spc loc (Option.call ~f:pro $ k)
-  | _ -> fun k -> Cmts.fmt c ~pro:spc ppat_loc (Option.call ~f:pro $ k) )
+  | _ ->
+      fun k ->
+        Cmts.fmt_before c ppat_loc
```

Comment:
Removing the `~pro` argument seems enough. If not, could you add an example to the tests?

Reply:
fixed